Mathlib/Analysis/RCLike/Basic.lean

```diff
@@ -239,6 +241,13 @@ instance (priority := 100) charZero_rclike : CharZero K :=
 lemma ofReal_expect {α : Type*} (s : Finset α) (f : α → ℝ) : 𝔼 i ∈ s, f i = 𝔼 i ∈ s, (f i : K) :=
   map_expect (algebraMap ..) ..
 
+@[norm_cast]
+lemma ofReal_balance {ι : Type*} [Fintype ι] (f : ι → ℝ) (i : ι) :
```

Comment:
```suggestion
lemma ofReal_balance_apply {ι : Type*} [Fintype ι] (f : ι → ℝ) (i : ι) :
```
optional

Reply:
Given that you can't take `ofReal` of `balance f` (but you can compose, cf `ofReal_comp_balance`), I'm going to say my name is the better name